reconcile/terraform_resources.py

```diff
@@ -220,6 +220,11 @@ def setup(
     # build a resource inventory for all the kube secrets managed by the
     # app-interface managed terraform resources
     tf_namespaces = get_tf_namespaces(account_names)
+    if not tf_namespaces:
+        logging.warning(
+            "No terraform namespaces found, consider disabling this integration, account names: "
+            f"{', '.join(account_names)}"
+        )
```

Comment:
should we also return/exit in this case?

Reply:
It's hard to do a return here as this is in an extracted setup function, and based on what I know, we'd like to remove usages of `sys.exit`.

Reply:
Normally this warning will be addressed soon after deploy and those accounts will be disabled, so even we don't stop here, it won't run anything in the end.